messaging/3.1/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Jakarta Messaging 3.1 (Under Development)"
+date: 2021-10-15
+summary: "Jakarta EE 10 Release"
+---
+
+Jakarta Messaging describes a means for Java applications to create, send, and receive messages via loosely coupled, reliable asynchronous communication services.
+
+# Plan
+Jakarta Messaging 3.1 will target the Jakarta EE 10 platform release. 
+
+Jakarta Messaging 3.1 will be a minor update with fixes and enhancements. 
+
+It will contain:
+
+* Bug fixes as they arise during the release cycle
+* Small enhancements to take advantage of some newer Java features in the lowest supported Java version (for example repeatable annotations)
+* A standalone TCK separated from the Jakarta EE TCK so that it's easier to certify standalone implementations of Jakarta Messaging
+* Documentation clean up as identified during the release cycle
+* Any updates required to meet the Java version requirements of Jakarta EE 10
+* Any requirements identified by other specifications or the Jakarta EE 10 platform projects during the release cycle.
+
+
+[Jakarta Messaging 3.1 Release Record](https://projects.eclipse.org/projects/ee4j.jca/releases/2.1.0)
```

Comment:
Incorrect copy-and-paste?  :-)  This should point at the JMS 3.1 release record, not the JCA 2.1 release record.

Reply:
Thanks for spotting this, @kwsutter. I've fixed it now.

Of course a copy and paste error, I created the release record before I submitted the PR but let the link from the JCA plan which I used as a template.